### DISCUSSION TITLE
/chat/new route send message is not working

### DISCUSSION THREAD

**Discussion by ido777** [danny-avila/LibreChat#1959] [Q&A] [answered]
### What happened?

1. In  http://localhost:3080/chat/new I clicked on this first example and then submitted nothing happened. I expected it to send the message.
2.   In  http://localhost:3080/chat/new I clicked on this second example nothing happened. I expected it to fill out the chat form, but nothing happened. 
3. I wanted to add an attachment clicked the symbol - it opened a different menu 

Regardless it seems that this route is using a different React comp than /c/new (therefore the different functionality) 0 can they be united? 

### Steps to Reproduce

Bug 1: Submitting does nothing

1. Open your web browser and navigate to /chat/new (e.g.  http://localhost:3080/chat/new) This should take you to the new chat message creation page.

1. Click on the first example. This action fill in the chat form with the example's content 

1.  click on the 'Submit' button. This is expected to send the message or show an error if the message cannot be sent or the form is incorrectly filled.

Bug 2: Clicking the second example does nothing
follow the same steps for the second example 

Bug 3: Incorrect menu opens when trying to add an attachment
Click on the logo at the left

Bug 4: go to http://localhost:3080/c/new and compare the functionality and the look and feel of this page.

### What browsers are you seeing the problem on?

Chrome (Win11) 

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment, marked as answer]
`/chat/new` is deprecated for a couple of versions now, and will be completely removed soon.

**Reply by JamieWuJinYi** [reply]
![image](https://github.com/danny-avila/LibreChat/assets/100826652/d66c89a8-6e7e-450a-b86e-fe0507670569)
Has this route been removed in the latest version 0.7.0? Looking forward to your reply.

@danny-avila

**Reply by danny-avila** [reply]
Yes, every so often I make even more changes that breaks the route further. It's not intended for use anymore.
